modules/nf-core/gemma/kinshipmatrix/meta.yml

```
@@ -0,0 +1,64 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "gemma_kinshipmatrix"
+description: GEMMA is a software toolkit for fast application of linear mixed models (LMMs)
+keywords:
+  - matrix
+  - lmms
+  - gwas
+tools:
+  - "gemma":
+      description: "Linear mixed models (LMMs) for genome-wide association (GWA)"
+      homepage: "https://github.com/genetics-statistics/GEMMA"
+      documentation: "https://github.com/genetics-statistics/GEMMA"
+      tool_dev_url: "https://github.com/genetics-statistics/GEMMA"
+      doi: "https://doi.org/10.1038/ng.2310"
```

```suggestion
      doi: "10.1038/ng.2310"
```
to satisfy the linter